iChef-WebApi/src/main/java/com/kitchen/iChef/Repository/Interfaces/IUserRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.kitchen.iChef.Repository.Interfaces;
+
+import com.kitchen.iChef.Domain.AppUser;
+import org.springframework.context.annotation.Profile;
+import org.springframework.data.repository.CrudRepository;
+
+
+@Profile("!test")
```

Comment:
I am not sure what this is, can you explain briefly? It jumped at me because of the string test which indicates to me that this may be left here by mistake

Reply:
its for different environments (made possible in future builds, right now its just to set a standard, might change later)